README.md

```diff
@@ -52,6 +52,30 @@ Least one is expected to check are the following configuration parameters
 }
 ```
 
+To enable network status updates from the driver one must take Global EventQueue into use via app config file. Otherwise
```

Comment:
This is not true..
`mbed_event_queue();` will spawn a thread UNLESS `"events.shared-dispatch-from-application": true`

So this setting is just a possible memory optimization..

Reply:
Maybe we should remove this whole section.. Let the event queue spawn its own thread.. allow those who try to optimize memory to study this from Mbed OS documentation.

Reply:
Section dropped.